pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -265,6 +288,58 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	return nil
 }
 
+func (c *Reconciler) reconcileMetricsService(ctx context.Context, pa *pav1alpha1.PodAutoscaler) error {
+	logger := logging.FromContext(ctx)
+
+	scale, err := c.kpaScaler.GetScale(ctx, pa)
+	if err != nil {
+		logger.Errorw("Error Getting scale", zap.Error(err))
```

Comment:
Do we need all the Error logging in this function? Should we rather `errors.Wrap` the errors and add context that way? We're logging the overall error anyway.

Reply:
Degraded to debug, so it won't show up in the production.

Reply:
Hm, but why even log on Debug? Doesn't the caller log it anyway?

Reply:
Easier to trace the origin of the error. I can go over in the very end to remove those (I usually annotate those with "###" to make sure I remove them in the end).